test/command/main-esm.test.ts

```diff
@@ -1,9 +1,8 @@
+import {runCommand} from '@oclif/test'
 import {expect} from 'chai'
 import {resolve} from 'node:path'
 import {pathToFileURL} from 'node:url'
 
-import {runCommand} from '../test'
```

Comment:
why do this change? @cristiand391 and I struggled to reverse this in `jsforce`?

Reply:
@WillieRuemmele I'm not sure what you mean. This change is to use the new major of oclif/test (which I originally wrote as POC inside `../test`). Not sure how it relates to jsforce